zio/anyio/mime.go

```diff
@@ -0,0 +1,74 @@
+package anyio
```

Comment:
I am hesitant about putting this in server since client packages will need access to this code as well and should probably not be depending on this package...

Reply:
Moved to package `api`.